test/integration/api/sessions.js

```diff
@@ -339,6 +341,15 @@ describe('api: /sessions', () => {
           .set('X-Forwarded-Proto', 'https')
           .set('Cookie', 'session=' + body.token)
           .expect(200))));
+
+    it('should succeed it is ajax/xhr request', testService((service) =>
+      service.authenticateUser('alice', 'includeCsrf')
```

Comment:
I see that the existing test right above does the same thing, but I don't totally understand passing the string `'includeCsrf'` to `authenticateUser()` instead of `true`. It looks to me like `authenticateUser()` is just checking whether that argument is truthy: it's not using it as a string.